chrome/background.js

```diff
@@ -1,16 +1,134 @@
-let languages = ['english', 'spanish', 'french', 'portuguese', 'russian']
+let glossaries = {
+	'english': 'english-terminology-for-editors',
+	'spanish': 'spanish-glossary',
+	'french': 'french-glossary',
+	'portuguese': 'portuguese-glossary',
+	'russian': 'russian-glossary'
+}
 
-chrome.runtime.onInstalled.addListener(function() {
+chrome.runtime.onInstalled.addListener(init)
+chrome.runtime.onStartup.addListener(init)
 
-	for (let i in languages) {
-		let lang = languages[i]
-		let glossaryName = lang + '-glossary'
-		const url = chrome.runtime.getURL('data/' + glossaryName + '.json');
+
+function init() {
+	getGlossaries().catch(() => {
+		console.log('Could not get ForumBee. Using local glossary instead.')
+		for (let lang in glossaries) {
+			let glossaryName = lang + '-glossary'
```

Comment:
@kevinzwang Isn't glossaries defined above as language-glossary already, so then this append here will end up creating something like french-glossary-glossary?

Reply:
No, the keys for the `glossaries` object is only the language name